pycozmo/window.py

```diff
@@ -5,6 +5,7 @@
 """
 
 import math
+from threading import Lock
```

Comment:
I agree, I also stopped liking to share the windows between send and receive threads after I started thinking on the thread safety.
I am thinking that, since the send thread does not need to look into the receive window, we could pass the send thread object as a member of receive thread, and keep the windows as they were before, and add a method in the send thread to do the needed interactions with the send window safely (as I think you mentioned somewhere).